.azure/bicep/uat.bicep

```diff
@@ -0,0 +1,44 @@
+param location string = resourceGroup().location
+
+var env = 'uat'
+
+var appHostnames = [
+  // TODO when a real prod environment is added, we should subdomain UAT
+  'hutch.com'
```

Comment:
remove all these? we're unlikely to host this on a custom domain name, but even if we do it'd be more likely `test.hutch.nottingham.ac.uk` or similar rather than `hutch.com` ;)